src/createTokenBridgePrepareTransactionRequest.ts

```diff
@@ -0,0 +1,92 @@
+import { Address, PublicClient, encodeFunctionData } from 'viem';
+
+import { tokenBridgeCreator } from './contracts';
+import { createTokenBridgeDefaultGasLimit, createTokenBridgeDefaultValue } from './constants';
+import { validParentChainId } from './types/ParentChain';
+import { createTokenBridgeGetInputs } from './createTokenBridge-ethers';
+import { publicClientToProvider } from './ethers-compat/publicClientToProvider';
+import { isCustomFeeTokenChain } from './utils/isCustomFeeTokenChain';
+
+type GasOverrideOptions = {
+  minimum?: bigint;
+  percentIncrease?: bigint;
+};
+
+export type TransactionRequestGasOverrides = {
+  gasLimit?: GasOverrideOptions;
+  retryableTicketFees?: GasOverrideOptions;
+};
```

Comment:
I only added these overrides for now, because these are the ones we need. In the new PR I add one more and move this type to the `types` folder since it's then used in 2 files.